### ISSUE TITLE
对argmax层的维度识别错误

### ISSUE THREAD

**Issue by Egozjuer** [alibaba/MNN#223]
如题，caffe中的prototxt的argmax层axis为1,转换成mnn格式后，在MNN架构下运行，在CPUArgMax.cpp添加打印，发现argmax层的前一层维度可以正确输入，但argmax层中的width变为了1，channel保持不变，尝试改变prototxt中axis，重新转换mnn，发现无论更换为0,1,2还是3,argmax的输出结果唯独都不变化，想请教下这是什么原因导致的啊

**Comment by WenguoLi** [comment]
tf.argmax(logits, 3, output_type=tf.int32) have the same problem,  
Looking forward to this bug being resolved as soon as possible.
Thanks

**Comment by li-qing** [comment]
fixing

**Comment by mengdong824** [comment]
> fixing

pull最新的代码，还是argmax层维度识别错误

**Comment by daikankan** [comment]
MNN-0.2.0.7发现同样的问题，argmax层中的width变为1，请问已经修复了吗？

**Comment by mengdong824** [comment]
我是把最后一层的argmax层单独拿出来，自己写个函数实现argmax的功能


------------------
Vehicle Engineering , South China University of Technology


余蒙


Cell:155-2106-4363


E-mail:ymeng825@foxmail.com


 




------------------ Original ------------------
